faq/admin.py

```diff
@@ -1,15 +1,40 @@
 from django.contrib import admin
+from django.utils.translation import gettext_lazy as _
 
+from util import html_utils
+from util.admin_utils import link_to_admin_change_form, search_escaped_and_unescaped
 from web.multilingual.admin import MultiLingualFieldAdmin
 from .models import Category, Question
 
 
 class QuestionAdmin(MultiLingualFieldAdmin):
+    list_display = ('title', 'get_categries', 'last_modified')
+    list_filter = ('categories',)
+    search_fields = ('title', 'answer', 'categories__name')
```

Comment:
Skal det være to understreker her? 
Også get_categries på linje 11.

Reply:
Yeh, `categories__name` betyr at man søker på `name` feltet i `Category` modellen (gjennom `categories`-`ManyToMany`-feltet) 😄 

Bra at du fiksa opp `get_categries` 😊 (oppdaterte også et felt du hadde glemt, i 662c4e50c928f7991e385410f537619bc76026b7)